packages/manager/modules/vps/src/migration/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,23 @@
+{
+  "vps_migration_title": "Migration VPS",
+  "vps_migration_button": "Programmer la migration",
+  "vps_migration_description2": "Votre VPS actuel va être remplacé par une machine de notre ",
+  "vps_migration_description2_new_range":"novelle gamme",
+  "vps_migration_description2_tail": ", à un prix inférieur ou équivalent.",
```

Comment:
Actually, we have a link in between the description 2. Hence, the text has been split into pre-link, link and post-link text, in order to avoid having html as a part of the translation. Please let us know your thoughts

Reply:
@ganeshkumar1989 I agree that HTML in the translations is not a better solution from our side but it would probably be easier for translations teams to translate a whole sentence than chunks that might not makes sense joined together in a sentence

Reply:
The `anchor` tag has been moved into the translation